llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -445,17 +446,18 @@ splitModule(Module &M,
   }
 }
 
-static std::string makeResultFileName(Twine Ext, int I) {
+static std::string makeResultFileName(Twine Ext, int I, bool IsEsimd) {
```

Comment:
I suggest to use `StringRef suffix` instead of `bool IsEsimd`, as filename generation uses this parameter only to create suffix. This would reduce 'IsEsimd' knowledge spread

Reply:
Done.